### DISCUSSION TITLE
Opendeck on my controllers and from customers!

### DISCUSSION THREAD

**Discussion by eurorackworld** [shanteacontrols/OpenDeck#214] [Show and tell]
I cannot state enough how much I like Opendeck and how I value all the work done for this. I'm using Opendeck currently on 99% on the controllers I'm selling (1% is just custom Arduino stuff for non midi boxes) and it's a breeze customizing the firmware to my needs and to give the customers the option to configure the controllers even better themselves.

I just started a few months ago but wanted to share a few of my projects here:
![PhotoRoom-20230614_212530](https://github.com/shanteacontrols/OpenDeck/assets/29734704/4c07907f-9337-47dd-99d5-abbb9c928c87)
![PhotoRoom-20230609_002630](https://github.com/shanteacontrols/OpenDeck/assets/29734704/acbcd816-61b8-49d2-8d8e-b748427e849c)
![PhotoRoom-20230527_095345](https://github.com/shanteacontrols/OpenDeck/assets/29734704/294477fa-13b5-4b48-a0a4-0cb588abe702)
![PhotoRoom-20230524_111630](https://github.com/shanteacontrols/OpenDeck/assets/29734704/d430a4da-3975-4119-b313-445db261776b)
![PhotoRoom-20230523_000838](https://github.com/shanteacontrols/OpenDeck/assets/29734704/ff015c58-65a3-4230-9623-1e87a4e6b32f)
![PhotoRoom-20230517_211039](https://github.com/shanteacontrols/OpenDeck/assets/29734704/90871dd4-bffb-4fef-8865-d5e567dd08ef)
![PhotoRoom-20230509_143947](https://github.com/shanteacontrols/OpenDeck/assets/29734704/36e111e6-8634-42b8-9ce6-7fcb373094da)
![PhotoRoom-20230503_150510](https://github.com/shanteacontrols/OpenDeck/assets/29734704/7f2003f5-a504-40ae-aa19-305dc876f7cc)
![PhotoRoom-20230502_162307](https://github.com/shanteacontrols/OpenDeck/assets/29734704/5683ebda-3b6f-44c9-9cb3-ac82cd832f29)
![photoroom-20230422_203725](https://github.com/shanteacontrols/OpenDeck/assets/29734704/1e801042-d44e-4053-9729-c58d876038fd)
![photoroom-20230420_110047](https://github.com/shanteacontrols/OpenDeck/assets/29734704/70121207-793e-4032-9b6f-ab5cb4571439)
![photoroom-20230420_105434](https://github.com/shanteacontrols/OpenDeck/assets/29734704/b1b6350f-3779-403f-94f4-17b7c379a5e9)
![photoroom-20230420_105251](https://github.com/shanteacontrols/OpenDeck/assets/29734704/564c85af-2e8a-4f0a-8126-57ae379a1610)
![photoroom-20230420_103938](https://github.com/shanteacontrols/OpenDeck/assets/29734704/a2cd5f71-b8ce-47b4-a176-9e93557b14fa)
<img width="1253" alt="Screenshot 2023-06-26 at 09 58 04" src="https://github.com/shanteacontrols/OpenDeck/assets/29734704/bc8b4a25-d6e3-4f15-8f62-77647382fe00">
<img width="1240" alt="Screenshot 2023-06-26 at 09 58 20" src="https://github.com/shanteacontrols/OpenDeck/assets/29734704/283de56a-49f4-44a0-9e0d-25112051fba8">

I'm also working on a big version for myself!
![2023-03-25 00 33 32](https://github.com/shanteacontrols/OpenDeck/assets/29734704/bc595215-2cc2-49e5-bd74-db8e9c199df5)
![2023-04-23 00 38 00](https://github.com/shanteacontrols/OpenDeck/assets/29734704/5c59dfb3-d342-42fe-8656-d6d30b9f0962)

And this is definitely not the end as there are so many ideas pending in my head and of course more advanced controllers with displays etc. But first wanted to focus on affordable cool small controllers!

**Comment by paradajz** [comment]
Amazing. Glad my work made it possible!

**Comment by eurorackworld** [comment]
![IMG_20231015_232331_759](https://github.com/shanteacontrols/OpenDeck/assets/29734704/1f08c3bb-b261-4922-8194-70e26da9ca7e)
![IMG_20230924_102411_337](https://github.com/shanteacontrols/OpenDeck/assets/29734704/7f6eec51-f711-4813-8215-0cf3dd3b3bcb)
![IMG_20230920_014538_252](https://github.com/shanteacontrols/OpenDeck/assets/29734704/933bd0e9-d415-42d0-a47b-9a1eff1ecdb0)
![IMG_20230919_132322_941](https://github.com/shanteacontrols/OpenDeck/assets/29734704/951c0eda-a365-4d39-bcc3-865e2092dbb3)
![IMG_20230916_182719_929](https://github.com/shanteacontrols/OpenDeck/assets/29734704/4f5af0fd-6cf7-4200-81f0-bc08fc3fc674)
![IMG_20230906_111751_272](https://github.com/shanteacontrols/OpenDeck/assets/29734704/99d022f4-048f-4be8-9b90-dc1a40730edd)
![2024-06-17 16 43-Photoroom copy 2](https://github.com/shanteacontrols/OpenDeck/assets/29734704/b92032e2-fdf6-4cf3-a040-6d6827258830)
